# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/crashtests/crashtests.list
# Commit: 1d7725b37869
# Full Hash: 1d7725b37869aa40919dfc4b3e0263ef35e1d7b9
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-07-08 16:33:59
# Description:
#   Bug 1902555 - Handle overflow in buffer size check in IsValidAudioData. r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215634
# ==============================================================================

diff -r 7c5450aa6e8e -r 1d7725b37869 dom/media/webcodecs/crashtests/crashtests.list
--- a/dom/media/webcodecs/crashtests/crashtests.list	Mon Jul 08 09:40:53 2024 +0000
+++ b/dom/media/webcodecs/crashtests/crashtests.list	Mon Jul 08 10:29:06 2024 +0000
@@ -6,3 +6,4 @@
 skip-if(Android) pref(dom.media.webcodecs.enabled,true) load 1889831.html
 skip-if(Android) pref(dom.media.webcodecs.enabled,true) pref(media.ffmpeg.encoder.enabled,true) load 1891117.html
 skip-if(Android) pref(dom.media.webcodecs.enabled,true) pref(media.ffmpeg.encoder.enabled,true) load 1897631.html
+skip-if(Android) pref(dom.media.webcodecs.enabled,true) load 1902555.html
